eq-author/src/App/SignInPage/CreateAccount/index.js

```diff
@@ -20,25 +20,56 @@ const CreateAccount = ({
   setCreateAccountFunction,
   setForgotPassword,
   errorMessage,
+  errorMessageEmail,
   setErrorMessage,
+  setErrorMessageEmail,
   setVerificationEmail,
 }) => {
   const [createEmail, setCreateEmail] = useState("");
   const [fullName, setFullName] = useState("");
   const [password, setPassword] = useState("");
   let errorRefCreateAcc = useRef();
+  const textValidEmailError = process.env.REACT_APP_ORGANISATION_ABBR
+    ? "Enter a valid " + process.env.REACT_APP_ORGANISATION_ABBR + " email"
+    : "Enter a valid authorised email";
```

Comment:
This looks really good, and works really well!

I've added a small change, based on the prototype.
```suggestion
  
  const textValidEmailError = process.env.REACT_APP_ORGANISATION_ABBR
    ? "Only " + process.env.REACT_APP_ORGANISATION_ABBR + " email addresses allowed"
    : "Enter a valid authorised email";
```

The first error banner's text in the prototype reads as "Only <Organisation> email addresses are allowed".